pkg/engine/source/filesystem.go

```diff
@@ -258,18 +258,31 @@ func checkQueryFeatureFlagDisabled(ctx context.Context, metadata map[string]inte
 
 	logger := logger.FromContext(ctx)
 	// Check if the rule is disabled via feature flag
-	disabled, err := queryParameters.FlagEvaluator.EvaluateWithOrgAndCustomVariables("k9-iac-disable-kics-rule", customVariables)
+	ruleIdDisabled, err := queryParameters.FlagEvaluator.EvaluateWithOrgAndCustomVariables("k9-iac-disable-kics-rule", customVariables)
 	if err != nil {
 		// If feature flag evaluation fails, log and continue (fail open)
-		logger.Warn().Err(err).Str("kics_id", kicsIDStr).Msg("Failed to evaluate feature flag for KICS rule")
+		logger.Warn().Err(err).Str("kics_id", kicsIDStr).Str("reason", "id").Msg("Failed to evaluate feature flag for KICS rule")
```

Comment:
Not sure what the reason here gives, I think we don’t need it for this error message.
Instead I’d like to add the ff that failed evaluation:
```suggestion
		logger.Warn().Err(err).Str("kics_id", kicsIDStr).Str("feature_flag", featureflags. IacDisableKicsRule).Msg("Failed to evaluate feature flag for KICS rule")
```